src/Sentry.Bindings.Cocoa/ApiDefinitions.cs

```diff
@@ -2293,4 +2253,48 @@ interface PrivateSentrySDKOnly
     [Static]
     [Export ("addReplayRedactClasses:")]
     void AddReplayRedactClasses (Class[] classes);
+
+    // +(NSDictionary<NSString *,id> * _Nullable)appStartMeasurementWithSpans;
+    [Static]
+    [NullAllowed, Export ("appStartMeasurementWithSpans")]
+    NSDictionary<NSString, NSObject> AppStartMeasurementWithSpans();
+
+    // +(SentryUser * _Nonnull)userWithDictionary:(NSDictionary * _Nonnull)dictionary;
+    [Static]
+    [Export ("userWithDictionary:")]
+    SentryUser UserWithDictionary (NSDictionary dictionary);
+
+    // +(SentryBreadcrumb * _Nonnull)breadcrumbWithDictionary:(NSDictionary * _Nonnull)dictionary;
+    [Static]
+    [Export ("breadcrumbWithDictionary:")]
+    SentryBreadcrumb BreadcrumbWithDictionary (NSDictionary dictionary);
+}
+
+// @interface SentryId : NSObject
+[BaseType (typeof(NSObject), Name = "_TtC6Sentry8SentryId")]
```

Comment:
FWIW this symbol doesn't exist in the native library since it was converted to Swift (same for the mangled name of `SentryIntegrationProtocol`). Shouldn't the generator catch that?

Reply:
@filipnavara we wish the generator was that smart... since the Cocoa APIs started transitioning to Swift, we've just been manually working around things that break the objective sharpie generated code in a [post build script](https://github.com/getsentry/sentry-dotnet/blob/00be6dff3083c0e292f723bbc9dd05e8917ed849/scripts/generate-cocoa-bindings.ps1#L132) (which is quite onerous to maintain). Definitely open to suggestions if you can think of a better way to do this!

Reply:
No suggestions at the moment, but it would be nice to fix the currently mismatched class names. It actively breaks some advanced scenarios like non-trimmed Debug apps on iOS with `Registrar=managed-static`. The Xamarin build creates a bridge code that tries to statically bind to the non-existing symbols and fails to load.

Reply:
Added issue https://github.com/getsentry/sentry-dotnet/issues/3469 to address